scripts/taco_to_burrito.py

```diff
@@ -32,25 +33,30 @@ def main():
             convert_markers(full_path, target_path)
 
 ################################################################################
-# Quick error printing logic that only prints the title of a section if there
-# are any errors in it. Could be handled better by returning errors from the
-# function but that is more work to implement then this is.
+# Quick error printing logic
 ################################################################################
-eprint_title: str = ""
-eprint_trigger: bool = False
-def set_eprint_title(title: str):
-    global eprint_title
-    global eprint_trigger
-    eprint_title = title
-    eprint_trigger = False
-def eprint(*args, **kwargs):
-    global eprint_trigger
-    if not eprint_trigger:
-        eprint_trigger = True
-        print(eprint_title)
-    print("   ", *args, **kwargs)
-
-
+error_location = ""
+class ErrorType(Enum):
+    INFO = 0
+    WARNING = 1
+    ERROR = 2
+    
+def eprint(*args):
+    error_level = ErrorType.WARNING
+    for arg in list(args):
+        if(type(arg) == ErrorType):
+            error_level = arg
+            
+    print(error_level.name + " in file \"" + error_location + "\": ")
```

Comment:
This is going to print out the file name for every single error. This ended up being a ton of unnecessary extra noise in the output. The previous version had logic to prevent this extra noise and only print out the file when the first error in that file was printed, that functionality should be preserved.

Reply:
in practice that doesn't happen: as far as i can see, the only time eprint is called is on the root node of the XML file. in the case of there being malformed XML, sure it might happen more than once in a file, but by that time the script has exited. i can't replicate a situation where there are multiple errors in one file, but i'll add a quick bool to check for this.